### ISSUE TITLE
Training result is not as good as the pre-trained model

### ISSUE THREAD

**Issue by nanma** [NVIDIA/tacotron2#102]
I train this Tacotron2 model with the latest version of code and hparams(commit:6e430).  After running 500 epochs, the validation loss 0.3308, but the alignment is not very good:
![image](https://user-images.githubusercontent.com/1190445/49358849-25782100-f70f-11e8-8ef1-18e09854d50b.png)

The training and validation loss curves are:
![image](https://user-images.githubusercontent.com/1190445/49359223-7ccac100-f710-11e8-9899-db071b8e4777.png)

![image](https://user-images.githubusercontent.com/1190445/49359188-53aa3080-f710-11e8-8b4b-f6f84bbde28d.png)

And the audio generated by my Tacotron2 model with the pre-trained Waveglow is not as good as the pre-trained Tacotron2 and Waveglow model.

Should I continue training with more than 500 epochs? Or any suggestion for improving training would be help.

**Comment by rafaelvalle** [comment]
Yes! Learning rate decay.

**Comment by dnnnew** [comment]
Hi @nanma ,

Is your alignment getting better or not?
Any update?

**Comment by nanma** [comment]
@dnnnew  
I train the model for more epochs with the default learning rate, the valid loss reaches 0.28+. After that I start to decay the lr and continue training. The valid loss is 0.27+ now, and the alignment is better than when only training 500 epochs.
The alignment is still not as good as the pre-trained model, I will continue training.

**Comment by beknazar** [comment]
> @dnnnew
> I train the model for more epochs with the default learning rate, the valid loss reaches 0.28+. After that I start to decay the lr and continue training. The valid loss is 0.27+ now, and the alignment is better than when only training 500 epochs.
> The alignment is still not as good as the pre-trained model, I will continue training.

@nanma Could you eventually reach the performance results of pre-trained models?

**Comment by anilgovind** [comment]
@dnnnew 
I am also interested to know about the performance of your trained data. Is it have the same performance of pre-trained model?
